Gems/ROS2/Code/Source/Lidar/LidarSensorConfiguration.cpp

```diff
@@ -73,6 +68,18 @@ namespace ROS2
         }
     }
 
+    LidarSensorConfiguration::LidarSensorConfiguration(AZStd::vector<LidarTemplate::LidarModel> availableModels)
+        : m_availableModels(AZStd::move(availableModels))
+    {
+        if (m_availableModels.empty())
+        {
+            return;
```

Comment:
Okay, good point. Warning will be fine here, unless it will be called with each engine start. If it shows every time, the your first approach must rather stay.

Reply:
It doesn't. Added a warning.